src/main/java/org/jboss/logmanager/ExtLogRecord.java

```diff
@@ -518,6 +520,15 @@ public void setThreadName(final String threadName) {
         this.threadName = threadName;
     }
 
+    /**
+     * The callers thread. Note this may be {@code null} if the record has been deserialized.
+     *
+     * @return the callers thread, or {@code null}
+     */
+    public Thread getCallerThread() {
+        return callerThread;
+    }
```

Comment:
I'd probably choose something other than "caller thread" because a log record doesn't have a "caller" per se... "original thread", or maybe just "thread"/"getThread" to be consistent with "getThreadID"? The JavaDoc of the log record "getThreadID" method is along the lines of "Get an identifier for the thread where the message originated" so maybe we'd say "the thread where the message originated".